settings.gradle.kts

```diff
@@ -60,7 +61,8 @@ include(
     ":opentelemetry-examples-zipkin",
     ":opentelemetry-examples-spring-native",
     ":opentelemetry-examples-kotlin-extension",
-    ":opentelemetry-examples-grpc"
+    ":opentelemetry-examples-grpc",
+    ":opentelemetry-examples-resource-detection-gcp"
```

Comment:
only 1 project is added in this PR

Reply:
Didn't quite understand this - I think you're referring to the two lines modified ?

Reply:
misread this - never mind :)